perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/ResourcesManagerBlImpl.java

```diff
@@ -783,6 +784,17 @@ public List<RichResource> getRichResources(PerunSession sess, Vo vo) {
 		return getResourcesManagerImpl().getRichResources(sess, vo);
 	}
 
+	@Override
+	public List<RichResource> getMailingServiceRichResourcesWithMember(PerunSession sess, Member member) {
+		AttributeDefinition ad;
+		try {
+			ad = getPerunBl().getAttributesManagerBl().getAttributeDefinition(sess, AttributesManager.NS_ENTITYLESS_ATTR_DEF + ":optOutMailingList");
+		} catch (AttributeNotExistsException e) {
+			throw new InternalErrorException(e);
```

Comment:
Maybe we can return an empty list instead of this exception, because the attribute definition is not created during Perun startup, thus it doesn't have to be used on every Perun instance. In such cases, we probably don't want the operation to fail.